src/paperless/auth.py

```diff
@@ -35,7 +35,6 @@ def authenticate(self, request):
             and request.headers["Referer"].startswith("http://localhost:4200/")
         ):
             user = User.objects.filter(is_staff=True).first()
-            print(f"Auto-Login with user {user}")
```

Comment:
Nit: should this be kept (either logged "properly" or just ignored)? I know its for debugging only